client/js/app/components/common/react_multi_select.js

```diff
@@ -0,0 +1,98 @@
+var React = require('react');
```

Comment:
Since we built the other react-select components on our own, I thought that was the pattern we were following - I honestly should have asked. I was also a bit wary about using `react-select` because I wondered if it meant that we should remove our own react-select code, and replace it all with react-select (which would have meant styling, too). I'm also at the point where I feel guilty throwing more dependencies into explorer, since the project is so big already. I really wanted to use https://github.com/instructure-react/react-select-box, but it hadn't been updated in a while and some of the styling didn't work well in the context we needed, so I ended up hacking the code there and throwing it into explorer.

In retrospect, I really should have asked #web what the correct decision should have been, and I'm kind of kicking myself for that.